docs/guides/integration-prebid-server-side.md

```diff
@@ -0,0 +1,415 @@
+
+Cookie:
+
+```
+__uid2_advertising_token=...advertising token...
+```
+
+Configuration:
+
+```
+pbjs.setConfig({
+    userSync: {
+        userIds: [{
+            name: 'uid2'
+        }]
+    }
+});
+```
+
+#### Server-Only Mode Value Example
+
+The following example sets the `value` field to an ID block containing the advertising token without storing it in a cookie.
+
+```
+pbjs.setConfig({
+    userSync: {
+        userIds: [{
+            name: 'uid2'
+            value: {
+                'uid2': {
+                    'id': '...advertising token...'
+                }
+            }
+        }]
+    }
+});
+```
+
+#### Passing a New Token: Server-Only Mode
+
+In server-only mode, since the prebid.js UID2 module receives only the advertising token, the token is only valid for a short period of time. For this reason, it is best to provide an advertising token on each page load.
+
+If needed, to determine if you need to provide a new token, see [Determining Whether the Module Has a Valid Token](#determining-whether-the-module-has-a-valid-token).
+
+## Prebid Implementation Notes and Tips
+
+In planning your Prebid implementation, consider the following:
+
+(**GWH_LP first two bullet points are a) original copy, which we need to simplify, b) my proposed rewrite. Any suggestions welcome.**)
+
+- (**ORIGINAL**): If you provide an expired identity, and the module has a valid identity which was refreshed from the identity you provide, the module uses the refreshed identity. The module stores the original token that it used for refreshing the token and uses the refreshed tokens as long as the original token matches the token that you provided. 
+
+- (**PROPOSED REVISION FROM GWH**): The module stores the original token provided to it, refreshes it as needed, and uses the refreshed token. If you provide an expired identity, and the module has a valid update from refreshing the same identity, the module uses the refreshed identity in place of the expired one you provided.
+
+   **ALSO SW suggests removing this bullet point altogether and said check with you (LP) (GWH)**.
+
+- If you provide a new token that doesn't match the original token used to generate any refreshed tokens, the module discards all stored tokens and uses the new token instead, and keeps it refreshed.
+
+- During integration testing, set `params.uid2ApiBase` to `"https://operator-integ.uidapi.com"`. You must set this value to the same environment (production or integration) that you use for generating tokens.
+
+- For an example of what a UID2 token might look like in the bid stream, when it's sent from an SSP to a DSP, see [What does a UID2 token look like in the bid stream?](../getting-started/gs-faqs.md#what-does-a-uid2-token-look-like-in-the-bid-stream) (**GWH_SW_LP_ Should this come out, per Andrei's comment?**)
```

Comment:
I think we can get rid of this. It's not needed for a guide, and they can go looking for the information if they really want to find it.

Reply:
Done, thx.